src/main/java/com/google/cloud/tools/managedcloudsdk/internal/download/DownloaderFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk.internal.download;
+
+import java.net.URL;
+import java.nio.file.Path;
+
+/** Downloader factory. */
+public class DownloaderFactory {
+
+  private final String userAgentString;
+
+  public DownloaderFactory(String userAgentString) {
+    this.userAgentString = userAgentString;
+  }
+
+  /**
+   * Get a new {@link Downloader} implementation.
```

Comment:
nit: s/Get/Returns/

Use a declarative verb-phrase instead of an imperative statement for summary fragments. http://go/java-style#s7.2-summary-fragment